src/server/index.jsx

```diff
@@ -288,19 +290,41 @@ server
         headers,
       });
 
+      let derivedPageType = 'Unknown';
+
       try {
-        const { service, isAmp, route, variant } = getRouteProps(
-          routes,
-          urlPath,
-        );
-        const data = await route.getInitialData({
+        const {
+          service,
+          isAmp,
+          route: { getInitialData, pageType },
+          variant,
+        } = getRouteProps(routes, urlPath);
+
+        // Set derivedPageType based on matched route
+        derivedPageType = pageType || derivedPageType;
```

Comment:
Also do we need both this and the defaulting here?: https://github.com/bbc/simorgh/pull/7058/files#diff-4b858acf3fefbe270de1ad3a6516dfb1R9

Reply:
The defaulting you linked to is for the status code. The CloudWatch metrics fail to push properly if the status code is not a string. So the defaulting of the status code in that function is to ensure the .`toString()` will always pass as there may be times when a status code is not passed in to the function.